### PR TITLE
Curlify more quote marks

### PR DESCRIPTION
**Pull request by tspiteri**
Curlify some double-quote marks missed in #435.

### CODE REVIEW

File: templates/learn/get-started.hbs [hunk 1]
```diff
@@ -74,7 +74,7 @@
     </header>
     <p>Let’s add a dependency to our application. You can find all sorts of libraries on <a href="https://crates.io"
         target="_blank" rel="noopener">crates.io</a>, the package registry for Rust. In Rust, we often refer to
-      packages as "crates".</p>
+      packages as “crates.”</p>
```
Comment (ashleygwilliams) [inline templates/learn/get-started.hbs]: could we put the quote before the `.` - thanks!

File: templates/learn/get-started.hbs [hunk 2]
```diff
@@ -154,8 +154,8 @@ fn main() {
       <div class="highlight"></div>
     </header>
     <p>Ferris (they/them) is the unofficial mascot of the Rust Community. Rust programmers call themselves
-      "Rustaceans", which is a play on the word "crustacean", which is set of creatures, one of which is a crab!</p>
-    <p>Ferris is a name playing off of the adjective, "ferrous", meaning of or pertaining to iron. Since Rust often
+      “Rustaceans,” which is a play on the word “crustacean,” which is set of creatures, one of which is a crab!</p>
```
Comment (ashleygwilliams) [inline templates/learn/get-started.hbs]: please put the quotes inside the `,`s - thanks!

File: templates/learn/get-started.hbs [hunk 3]
```diff
@@ -154,8 +154,8 @@ fn main() {
...
+    <p>Ferris is a name playing off of the adjective, “ferrous,” meaning of or pertaining to iron. Since Rust often
```
Comment (ashleygwilliams) [inline templates/learn/get-started.hbs]: quote inside comma, please!

File: templates/tools/install.hbs [hunk 4]
```diff
@@ -84,7 +84,7 @@
         which provides just the Visual C++ build tools. Alternately, you
         can <a href="https://www.visualstudio.com/downloads/" target="_blank" rel="noopener">install</a>
         Visual Studio 2017, Visual Studio 2015, or Visual Studio 2013 and during install select
-        the "C++ tools".
+        the “C++ tools.”
```
Comment (ashleygwilliams) [inline templates/tools/install.hbs]: here as well

**Comment by tspiteri** [comment]
I had the impression that in American English, commas and periods should always be inside the quotation marks ([source, section *Quotation Marks with Commas and Periods*](https://www.quickanddirtytips.com/education/grammar/how-to-use-quotation-marks)). So I “fixed” those as I changed straight quotes to curly quotes.

In fact I found three already existing instances of this, so if the commas/periods are to be left outside the quotes here, they should be moved outside in these three instances:

  * *templates/learn/index.hbs*: `Affectionately nicknamed “the book,” The Rust...`
  * *templates/learn/index.hbs*: `It’s also sometimes called “the ’nomicon.”`
  * *templates/policies/code-of-conduct.hbs*: `... the user will be “kicked,” i.e., kicked out...`

Incidentally, in PR #435 I also changed the order for one instance in *templates/what/embedded/showcase.hbs*; I changed:

`...with the tag line 'safe, fast, concurrent &ndash; pick three'.`

to:

`... with the tag line ‘safe, fast, concurrent &ndash; pick three.’`

**Question:** Which convention should be used?

**Comment by steveklabnik** [comment]
“Commas and periods inside” is the style for all documentation, at least. I don’t know how the website people feel about it.

**Comment by ashleygwilliams** [comment]
Ok @tspiteri - sorry about the confusion! Let's stick with what you originally had.

**Pull request merged**